bitnami/opensearch/values.yaml

```diff
@@ -0,0 +1,2685 @@
+      ##
+      minReplicas: 3
+      ## @param master.autoscaling.hpa.maxReplicas Maximum number of APISIX Data Plane replicas
+      ##
+      maxReplicas: 11
+      ## @param master.autoscaling.hpa.targetCPU Target CPU utilization percentage
+      ##
+      targetCPU: ""
+      ## @param master.autoscaling.hpa.targetMemory Target Memory utilization percentage
+      ##
+      targetMemory: ""
+
+## @section Data-only nodes parameters
+
+data:
+  ## @param data.replicaCount Number of data-only replicas to deploy
+  ##
+  replicaCount: 2
+
+  ## @param data.extraRoles Append extra roles to the node role
+  ##
+  extraRoles: []
+
+  ## Pod Disruption Budget configuration
+  ## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb
+  ## @param data.pdb.create Enable/disable a Pod Disruption Budget creation
+  ## @param data.pdb.minAvailable Minimum number/percentage of pods that should remain scheduled
+  ## @param data.pdb.maxUnavailable Maximum number/percentage of pods that may be made unavailable
+  ##
+  pdb:
+    create: false
+    minAvailable: 1
+    maxUnavailable: ""
+
+  ## @param data.nameOverride String to partially override opensearch.data.fullname
+  ##
+  nameOverride: ""
+  ## @param data.fullnameOverride String to fully override opensearch.data.fullname
+  ##
+  fullnameOverride: ""
+  ## @param data.servicenameOverride String to fully override opensearch.data.servicename
+  ##
+  servicenameOverride: ""
+  ## @param data.annotations [object] Annotations for the data statefulset
+  ##
+  annotations: {}
+  ## @param data.updateStrategy.type Data-only nodes statefulset stategy type
+  ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
+  ##
+  updateStrategy:
+    type: RollingUpdate
+  ## Opensearch resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## @param data.resources.limits The resources limits for the data containers
+  ## @param data.resources.requests [object] The requested resources for the data containers
+  ##
+  resources:
+    limits: {}
+    requests:
+      cpu: 25m
+      memory: 2048Mi
+  ## @param data.heapSize Opensearch data node heap size.
+  ## Note: The recommended heapSize is half of the container's memory.
+  ## If omitted, it will be automatically set.
+  ## Example:
+  ##   heapSize: 128m
+  ##
+  heapSize: 1024m
+  ## Configure Pods Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param data.podSecurityContext.enabled Enabled data pods' Security Context
+  ## @param data.podSecurityContext.fsGroup Set data pod's Security Context fsGroup
+  ##
+  podSecurityContext:
+    enabled: true
+    fsGroup: 1001
+  ## Configure Container Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
```

Comment:
The seccomp section is still missing

Reply:
[c8fb3ba](https://github.com/bitnami/charts/pull/17509/commits/c8fb3ba04bd41a90c8bfc2bf1b1d804c35582679)